source/includes/_notifications.md

```diff
@@ -724,4 +857,4 @@ Parameter | Type | Description
 --------- | ---- | -----------
 event_uri | string | The event_uri for the desired event.
 notification_id | integer | The id for the desired notification
-id | integer | The id for the notification language
\ No newline at end of file
+id | integer | The id for the notification language
```

Comment:
Id for the desired notification language

Reply:
Done